CHANGES.md

```diff
@@ -1,6 +1,14 @@
 ringpop-go changes
 ==================
 
+v0.DEV (to be released)
+-----------------------
+
+* Remove `File` and `Host`-based discover providers in favor of
```

Comment:
This is a breaking change, so I think we need to provide an additional section: "release notes" which gives clear instructions on how to migrate from the old API to the new. There is a "release notes" section further down in this readme to get an example from.